docs/guides/data-loading.md

```diff
@@ -105,7 +105,7 @@ Remix polyfills the `fetch` API on your server so it's very easy to fetch data f
 
 ```tsx filename=app/routes/gists.jsx lines=[2]
 export function loader() {
```

Comment:
If you are using `await`, then the function should have the `async` keyword.

```suggestion
export async function loader() {
```

Reply:
This is totally right!